### PR TITLE
feat: add rounding, cl, and comparison mutators

### PR DESCRIPTION
**Pull request by czarcas7ic**
Adds the following rounding mutations:
```
	"QuoTruncate":    "QuoRoundUp",
	"MulTruncate":    "MulRoundUp",
	"QuoRoundUp":     "QuoTruncate",
	"MulRoundUp":     "MulTruncate",
	"TruncateInt":    "RoundInt",
	"RoundInt":       "TruncateInt",
	"QuoRoundupMut":  "QuoTruncateMut",
	"QuoTruncateMut": "QuoRoundupMut",
```

Adds the following getter mutations:
```
	"GetToken0": "GetToken1",
	"GetToken1": "GetToken0",
```

Adds the following cl mutations:
```
	"Liquidity0":       "Liquidity1",
	"Liquidity1":       "Liquidity0",
	"CalcAmount0Delta": "CalcAmount1Delta",
	"CalcAmount1Delta": "CalcAmount0Delta",
```

### CODE REVIEW

File: mutator/cosmos/comparison.go [hunk 1]
```diff
@@ -12,8 +12,8 @@ func init() {
 }
 
 var comparisonMutations = map[string]string{
-	"GT": "LTE",
-	"LT": "GTE",
+	"GT":  "LTE",
+	"LT":  "GTE",
 	"GTE": "LT",
 	"LTE": "GT",
```
Comment (ValarDragon) [inline mutator/cosmos/comparison.go]: Can we either in this PR or a followup add logic to also test GT -> GTE & LT -> LTE.

(Need to make the map value field a slice I assume, and produce multiple mutations)

**Review by ValarDragon — APPROVED** [review]
Nice

File: mutator/cosmos/arithmetic.go [hunk 2]
```diff
@@ -12,10 +12,18 @@ func init() {
...
 
 var arithmeticMutations = map[string]string{
-	"Add": "Sub",
-	"Sub": "Add",
-	"Mul": "Quo",
-	"Quo": "Mul",
+	"Add":            "Sub",
+	"Sub":            "Add",
+	"Mul":            "Quo",
+	"Quo":            "Mul",
```
Comment (p0mvn) [inline mutator/cosmos/arithmetic.go:18]: Consider mutative variants such as `AddMut`

**Review by p0mvn — APPROVED** [review]
Nice, thanks for adding

File: mutator/cosmos/arithmetic.go (hunk 2 above)
Comment (czarcas7ic) [reply mutator/cosmos/arithmetic.go:18]: Added! https://github.com/osmosis-labs/go-mutesting/pull/9/commits/ecfd0960c406f803c97523e2faf635ea3c07310b

File: mutator/cosmos/comparison.go (hunk 1 above)
Comment (czarcas7ic) [reply mutator/cosmos/comparison.go]: Added here! https://github.com/osmosis-labs/go-mutesting/pull/9/commits/e6c5dd0adea030bf634d712c918b2cd3c16b130e

**Review by AlpinYukseloglu — APPROVED** [review]
Great job. If you want to fill in the rest from the list in https://github.com/osmosis-labs/osmosis/issues/5515 would be awesome but imo good to merge these ones and add the rest in a follow up

**Pull request merged**